packages/main/src/themes/Label.css

```diff
@@ -1,38 +1,49 @@
 :host(:not([hidden])) {
 	display: inline-flex;
 }
-
 :host {
 	max-width: 100%;
 	color: var(--sapUiContentLabelColor);
 	font-family: var(--sapUiFontFamily);
 	font-size: var(--sapMFontMediumSize);
 	font-weight: normal;
 	cursor: text;
+	direction: ltr;
```

Comment:
Better remove `direction: ltr;`.
 I know that text in LTR languages (such as Bulgarian, English, etc.) looks bad in RTL, but if you leave this `direction: ltr` hardcoded, then we brake Hebrew and similar RTL languages (for example: test with אני מבולגריה and set/unset this `direction: ltr` via Chrome Dev tools)

Note: To be able to edit the styles in the Dev tools, you might need to add the following script in the Label.html
```js
<script>delete Document.prototype.adoptedStyleSheets</script>
```